src/containers/table/Table.js

```diff
@@ -222,12 +253,21 @@ class TableView extends React.Component {
     });
   }
 
+  sortResult(result) {
+    if (this.state.sortBy == 'name') {
+      return changeOrder(result, this.state.sortBy, this.state.order);
+    } else {
+      return changeNumOrder(result, this.state.sortBy, this.state.order);
+    }
+  }
+
   render() {
     const filteredCompanies = filterCompanies(this.state.companies, this.state.selectedStages,
       this.state.selectedTags);
-    let searchResults = filteredCompanies.filter((comp) => {
+    let searchResultsUnSorted = filteredCompanies.filter((comp) => {
       return comp.name.toLowerCase().includes(this.state.search.toLowerCase());
     });
+    let searchResults = this.sortResult(searchResultsUnSorted);
```

Comment:
somewhere in the near future this part needs to get cleaned up, render function should ideally focus on rendering and not deal with filtering arrays.